data-prepper-plugins/ml-inference-processor/src/main/java/org/opensearch/dataprepper/plugins/ml_inference/processor/MLProcessorConfig.java

```diff
@@ -33,6 +33,9 @@
 public class MLProcessorConfig {
     private static final int DEFAULT_MAX_BATCH_SIZE = 100;
     public static final Duration DEFAULT_RETRY_WINDOW = Duration.ofMinutes(10);
+    public static final int DEFAULT_RETRY_INTERVAL = 60;    // default retry interval is 1 minute
+    private static final int MIN_RETRY_INTERVAL = 3;
+    public static final int MAX_RETRY_INTERVAL = 300;
```

Comment:
Nit: suggest specifying the time unit in the name, eg. DEFAULT_RETRY_INTERVAL_SECONDS, etc.